CourseApp/Calculation.cs

```diff
@@ -0,0 +1,28 @@
+namespace CourseApp
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class Calculation : MainFunction
```

Comment:
Please do not use inheritance here, composition or aggregation will be more suitable